### ISSUE TITLE
Including @semantic-release/error in plugins crashes

### ISSUE THREAD

**Issue by akselinurmio** [semantic-release/semantic-release#1274]
## Current behavior

semantic-release was working great until I wanted to add a plugin, and therefore had to create a `plugins` array in my configuration.

[From the docs](https://semantic-release.gitbook.io/semantic-release/usage/plugins#plugins-declaration-and-execution-order) I read that if you want to add a plugin, I will need to include all of the default plugins in the array as well. The documentation also [includes a list](https://semantic-release.gitbook.io/semantic-release/usage/plugins#plugins-installed-by-default) of plugins that are already installed. And that includes `@semantic-release/error`.

If I include that plugin to my config’s plugin array, the process throws the following exception:
```
[3:03:08 PM] [semantic-release] › ✖  EPLUGINSCONF The `plugins` configuration is invalid.
The plugins (https://github.com/semantic-release/semantic-release/blob/master/docs/usage/configuration.md#plugins) option must be an array of plugin definions. A plugin definition is an npm module name, optionnaly wrapped in an array with an object.

The invalid configuration is [Function: SemanticReleaseError].
```

Which isn’t really helpful. I spent a good time trying to find the cause.

## Expected behavior

I would guess that the documentation shouldn’t tell people to include the `@semantic-release/error` plugin in their configurations.

## Environment

- **semantic-release** version: 15.13.21
- CI environment: travis-ci
- Plugins used: 
- **semantic-release** configuration: https://github.com/akselinurmio/virtuaaliviivakoodi/blob/138b0bbd4d1f96a0fa7684464aa032032a5a240f/package.json#L51-L65
- CI logs: https://travis-ci.org/akselinurmio/virtuaaliviivakoodi/builds/575292315

**Comment by pvdlg** [comment]
`@semantic-release/error` is not a plugin and the doc is incorrect. Will be fixed by #1275.

**Comment by akselinurmio** [comment]
Great to hear! 👌
